requirements.txt

```diff
@@ -13,5 +13,6 @@ pyside2
 qutip
 scipy
 shapely  #using conda on Windows OS: pip install of shapely will fail on import for missing geos_c.dll, if <= v1.7.x. For as long as a 1.8.0 version is not released (already patched in github), you will need to run `conda install shapely` (before installing qiskit_metal).
+git+git://github.com/scqubits/scqubits.git@jk-explorer # until scqubits main branch supports come online
```

Comment:
sorry, but cannot do this. Pypi deploy require for no dependency from github forks/repo. only other pypi packages can be referenced.